setup/mysql.sql

```diff
@@ -145,10 +147,25 @@ INSERT INTO `vexim`.`users`
     smtp, pop, realname, type, admin
 )
 VALUES 
-(   '1', 'siteadmin', 'siteadmin', '$1$12345678$2lQK5REWxaFyGz.p/dos3/', '65535', '65535', '', '',  'SiteAdmin', 'site', '1'
+(   '1', 'siteadmin', 'siteadmin', '$6$uR.EiB1dj5rrvwMF$Qh5LgdjOZavKXwhi9IF0Yuzu7qxsG.dLTTB8e./55ZRNfBuZVLnfUSOEXa0oWT6174myO.WYkOy83HYWAKPbK/', '65535', '65535', '', '',  'SiteAdmin', 'site', '1'
```

Comment:
My point was, it's not a good idea to keep the default password in a production system anyway. `$1$` works in all setups, while `$6$` might not (at least to my understanding, correct me if I'm wrong). It doesn't matter which encoding scheme we use in our default sql file, because everyone knows what the password behind that is anyway. I would actually opt for keeping it in plain text, if it worked, but it doesn't. :)